### PR TITLE
Display victim address

### PR DESCRIPTION
**Pull request by rottencandy**
Displays victim address by querying Mapbox using lat, long data.

### CODE REVIEW

File: src/Display.tsx [hunk 1]
```diff
@@ -14,6 +14,19 @@ interface VictimDetailProps {
...
+  const [address, setAddress] = useState("");
+  fetch(
+    `https://api.mapbox.com/geocoding/v5/mapbox.places/${props.data.lat},${props.data.lon}.json?` +
+      new URLSearchParams({
+        access_token: process.env.REACT_APP_MAPBOX_TOKEN || ""
+      })
+  )
+    .then(response => response.json())
+    .then(jsonData => {
+      if (jsonData.features.length > 0) {
```
Comment (invincibleJai) [inline src/Display.tsx]: just `jsonData.features.length` would be good

File: src/Display.tsx [hunk 2]
```diff
@@ -14,6 +14,19 @@ interface VictimDetailProps {
 }
 
 const VictimDetail: React.FC<VictimDetailProps> = props => {
+  const [address, setAddress] = useState("");
+  fetch(
+    `https://api.mapbox.com/geocoding/v5/mapbox.places/${props.data.lat},${props.data.lon}.json?` +
```
Comment (invincibleJai) [inline src/Display.tsx]: can we have `host` as constant

Comment (rottencandy) [reply src/Display.tsx]: > can we have `host` as constant

Should I store API url in const var instead of directly using string?

Comment (rottencandy) [reply src/Display.tsx]: Also, would it be better to use [`URL()`](https://developer.mozilla.org/en-US/docs/Web/API/URL/URL) instead?

File: Dockerfile [hunk 3]
```diff
@@ -3,8 +3,7 @@ FROM node:12-alpine
 COPY . .
 
 RUN npm install && \
-    npm run build && \
-    npm install -g serve
+    npm run build
```
Comment (invincibleJai) [inline Dockerfile]: what's the point of `npm run build` if you are using `npm run start`

Comment (rottencandy) [reply Dockerfile]: When the container starts, the pipeline sets env vars which contain urls for the backend service & ERD services. `npm run build` seems to be the only way to make npm read env vars during container start.

**Pull request merged**